python/paddle/v2/framework/tests/test_network.py

```diff
@@ -0,0 +1,25 @@
+from paddle.v2.framework.network import Network
+import unittest
+
+
+class TestNetwork(unittest.TestCase):
+    def test_all(self):
+        net = Network()
+        out = net.add_two(X="A", Y="B")
```

Comment:
Here, I'm entangled in how to expose the API better.

@Canpio @Superjom 

Should our network expose

```python
net.add_two(...)
net.mul(...)
```

or just simple

```
net.create_and_add_op("add_two", ...)
net.create_and_add_op("mul", ...)
```

The second way will avoid dynamic method generating by `__getattr__`, but make user write more characters. Also, caffe2, tf, keras are using `mul(...)`.

Personally, I think `create_and_add_op('mul', ...)` is better because it make our code simpler. 

There is a similar issue before #3198 . So, should we change to `create_and_add_op("mul", ...)` ?